lib/src/mustachio/parser.dart

```diff
@@ -490,13 +510,16 @@ class _KeyParseResult {
 
   final List<String> names;
 
-  const _KeyParseResult._(this.type, this.names);
+  final SourceSpan /*?*/ span;
```

Comment:
I know it's an internal class, but I think a doc comment here specifying at least what cases it could be null in would increase the future readability and usability of it.

Reply:
Fair point! Documented